casepro/backend/junebug.py

```diff
@@ -115,7 +190,28 @@ def pull_contacts(
             tuple of the number of contacts created, updated, deleted and
             ignored
         """
-        return (0, 0, 0, 0)
+        identity_store = self.identity_store
+
+        # all identities created in the Identity Store in the time window
+        new_identities = identity_store.get_identities(
+            created_from=modified_after, created_to=modified_before)
+
+        # all identities modified in the Identity Store in the time window
+        modified_identities = identity_store.get_identities(
+            updated_from=modified_after,
+            updated_to=modified_before)
+
+        identities_to_update = list(chain(modified_identities, new_identities))
+
+        # all identities deleted in the Identity Store in the time window
+        deleted_identities = list(identity_store.get_identities(
+            optout_type='forget', updated_from=modified_after,
```

Comment:
Ah, I see that this is a specified filter, not a django query. Ignore that comment.